src/main/java/org/alliancegenome/curation_api/jobs/BulkLoadJobExecutor.java

```diff
@@ -187,107 +164,84 @@ public void process(BulkLoadFile bulkLoadFile) throws Exception {
 					config.setLoadOnlyIRIPrefix("MP");
 					processTerms(bulkLoadFile, mpTermService, config);
-					break;
-				case RO:
+				}
+				case RO ->
 					// config.setLoadOnlyIRIPrefix("RO");
 					processTerms(bulkLoadFile, roTermService, config);
-					break;
-				case MA:
-					processTerms(bulkLoadFile, maTermService, config);
-					break;
-				case WBBT:
-					processTerms(bulkLoadFile, wbbtTermService, config);
-					break;
-				case DAO:
+				case MA -> processTerms(bulkLoadFile, maTermService, config);
+				case WBBT -> processTerms(bulkLoadFile, wbbtTermService, config);
+				case DAO -> {
 					config.setLoadOnlyIRIPrefix("FBbt");
 					processTerms(bulkLoadFile, daoTermService, config);
-					break;
-				case WBLS:
-					processTerms(bulkLoadFile, wblsTermService, config);
-					break;
-				case FBDV:
-					processTerms(bulkLoadFile, fbdvTermService, config);
-					break;
-				case MMUSDV:
+				}
+				case WBLS -> processTerms(bulkLoadFile, wblsTermService, config);
+				case FBDV -> processTerms(bulkLoadFile, fbdvTermService, config);
+				case MMUSDV -> {
 					config.getAltNameSpaces().add("mouse_developmental_stage");
 					config.getAltNameSpaces().add("mouse_stages_ontology");
 					processTerms(bulkLoadFile, mmusdvTermService, config);
-					break;
-				case ZFS:
-					processTerms(bulkLoadFile, zfsTermService, config);
-					break;
-				case XBA_XBS:
+				}
+				case ZFS -> processTerms(bulkLoadFile, zfsTermService, config);
+				case XBA_XBS -> {
 					config.getAltNameSpaces().add("xenopus_anatomy");
 					config.getAltNameSpaces().add("xenopus_anatomy_in_vitro");
 					processTerms(bulkLoadFile, OntologyBulkLoadType.XBA, xbaTermService, config);
 					GenericOntologyLoadConfig config2 = new GenericOntologyLoadConfig();
 					config2.getAltNameSpaces().add("xenopus_developmental_stage");
 					processTerms(bulkLoadFile, OntologyBulkLoadType.XBS, xbsTermService, config2);
-					break;
-				case XPO:
+				}
+				case XPO -> {
 					config.setLoadOnlyIRIPrefix("XPO");
 					processTerms(bulkLoadFile, xpoTermService, config);
-					break;
-				case ATP:
+				}
+				case ATP -> {
 					config.setLoadOnlyIRIPrefix("ATP");
 					processTerms(bulkLoadFile, atpTermService, config);
-					break;
-				case XBED:
-					processTerms(bulkLoadFile, xbedTermService, config);
-					break;
-				case XSMO:
-					processTerms(bulkLoadFile, xsmoTermService, config);
-					break;
-				case OBI:
+				}
+				case XBED -> processTerms(bulkLoadFile, xbedTermService, config);
+				case XSMO -> processTerms(bulkLoadFile, xsmoTermService, config);
+				case OBI -> {
 					config.setLoadOnlyIRIPrefix("OBI");
 					processTerms(bulkLoadFile, obiTermService, config);
-					break;
-				case WBPheno:
-					processTerms(bulkLoadFile, wbPhenotypeTermService, config);
-					break;
-				default:
+				}
+				case WBPheno -> processTerms(bulkLoadFile, wbPhenotypeTermService, config);
+				case PATO -> processTerms(bulkLoadFile, patoTermService, config);
```

Comment:
This config needs to match the config in PatoTermCrudController

Reply:
If you do set a prefix it needs to be set here also.

Reply:
Ok, removed the prefix as there is a default namespace 'quality' in the file.